src/logic/calculate.js

```diff
@@ -0,0 +1,23 @@
+const calculate = (dataObject, simbol) => {
+  const { total } = dataObject;
+  const { next } = dataObject;
+  let { operation } = dataObject;
+
+  switch (simbol) {
+    case '+/-':
+      operation = total * (-1);
+      break;
+    case '-':
+      operation = total - next;
+      break;
+    case '+':
+      operation = total + next;
+      break;
+    default:
+      operation = total / next;
+      break;
+  }
+  return operation;
```

Comment:
- [ ] Instead of returning only this variable, return the whole object, like I explained above.